exporters/prometheus/src/test/java/io/opentelemetry/exporters/prometheus/PrometheusCollectorTest.java

```diff
@@ -68,7 +68,7 @@ void registerToDefault() throws IOException {
     return ImmutableList.of(
         MetricData.create(
             Descriptor.create(
-                "name",
+                "grpc.name",
```

Comment:
Is this the idiomatic name for these metrics (so before we automatically prefixed grpc, now users do)? If not, it might be clearer to have the tests match what we expect users should be doing as some free documentation.

Reply:
None of the metric semantic conventions for these things have landed yet. So, not sure at this point. This PR doesn't change what was there before, but it's a good question, and we should definitely make the tests match the semantic conventions when they exist.